usaspending_api/awards/v2/filters/transaction.py

```diff
@@ -171,6 +171,7 @@ def transaction_filter(filters):
                     awarding_agency__subtier_agency__name__in=awarding_subtier
                 )
 
+        # TODO update this to recipient_search_text if we ever switch back from the matview
```

Comment:
AFAIK this is still used by bulk downloader, it isn't dead code yet

Reply:
We should update this here now as well